packages/tests/package.json

```diff
@@ -55,6 +55,7 @@
     "@waku/core": "*",
     "@waku/enr": "*",
     "@waku/interfaces": "*",
+    "@waku/message-hash": "^0.1.17",
```

Comment:
nit: let's merge this PR after https://github.com/waku-org/js-waku/pull/2399 so that you can consume `messageHash` from `core`.